src/drivers/indexeddb.js

```diff
@@ -994,42 +1019,49 @@ function dropInstance(options, callback) {
                     return;
                 }
 
-                var newVersion = db.version + 1;
+                const newVersion = db.version + 1;
 
                 _deferReadiness(options);
 
-                var dbContext = dbContexts[options.name];
-                var forages = dbContext.forages;
+                const dbContext = dbContexts[options.name];
+                const forages = dbContext.forages;
 
-                for (var i = 0; i < forages.length; i++) {
-                    var forage = forages[i];
-                    if (forage._dbInfo.db) {
-                        forage._dbInfo.db.close();
-                        forage._dbInfo.db = null;
-                        forage._dbInfo.version = newVersion;
-                    }
+                db.close();
+                for (let i = 0; i < forages.length; i++) {
```

Comment:
I don't think we should be using `let` inside `for` loops, [because JS is weird](https://stackoverflow.com/questions/16473350/let-keyword-in-the-for-loop) though the bugs there are kind of fixed. But we aren't re-assigning `i` at any rate for each loop, so `const` still makes more sense.

Reply:
Oh my, browser vendors are weird

Reply:
But, [const  isn't allowed in for loops](https://babeljs.io/repl/#?babili=false&browsers=IE%208&build=&builtIns=usage&code_lz=GYVwdgxgLglg9mABMAFDAlIg3gKEYiBAZzgBsBTAOlLgHM10BuHAXxx0LCKkQC9EAvIgDaAcgCGogDSiARtNERRAXWbA4AJ0QpO3RDEGIADI32IAPH2rkwtKAAtTMANTPMuAJBFyUACowAW3I4ECgUFEwBAD5kBiZWHCA&debug=false&forceAllTransforms=false&shippedProposals=false&circleciRepo=&evaluate=false&fileSize=true&lineWrap=true&presets=es2015-loose&prettier=false&targets=&version=6.26.0&envVersion=1.6.2) and `let` will be transpiled anyway.
What do you suggest?

Reply:
Oh shoot, I realised that, we should do `let`. My brain got confused because in `for... in/of` loops it makes sense to use `const` I think, which we do in the Mozilla addons JS. This is fine 😄